README.md

```diff
@@ -0,0 +1,15 @@
+# RustPython parser as a library
+
+This project is forked from [RustPython][RustPython] project. The parser one of the core part of [RustPython] and [Ruff] project.
```

Comment:
Nit: missing "is" between "parser" and "one"

Reply:
Thanks!